fuzzers/fuzzer.mk

```diff
@@ -1,17 +1,27 @@
 N ?= 1
 SPECIMENS := $(addprefix build/specimen_,$(shell seq -f '%03.0f' $(N)))
 SPECIMENS_OK := $(addsuffix /OK,$(SPECIMENS))
+ENV_VAR ?=
+VAR ?=
+ITER ?= 0
+MAX_ITER ?= 10
 
 all: database
 
 $(SPECIMENS_OK):
 	mkdir -p build
-	bash ${XRAY_DIR}/utils/top_generate.sh $(subst /OK,,$@)
+	if [ -f `pwd`/generate.sh ]; then export $(ENV_VAR); bash `pwd`/generate.sh $(subst /OK,,$@); else bash ${XRAY_DIR}/utils/top_generate.sh $(subst /OK,,$@); fi
 
```

Comment:
FUZDIR over pwd?

Reply:
They should behave in the same way, probably it's better to stick with FUZDIR for coherency